docs/reference.md

```diff
@@ -1826,35 +1828,59 @@ Allowed values:
 
 Executes an HTTP request.
 
-_nb: Only APIs that return a JSON response are supported.
-
 Key | Type | Required |
 ------------ | -------| --- |
 [url](#url) | String | Yes |
 [method](#method) | String | Yes |
-[body](#body) | String | No |
+[body](#body) | Object/String | No |
 [headers](#headers) | String | No |
 
+_nb: For multipart/form-data content type requests, the body should be provided as a key/value object. For other content types, it can be provided as String in JSON format.
+
+Output | Type |
+----|----|
+body | String containing the response body
+status | Response status code
+
 Example:
 ```yaml
 activities:
   - execute-request:
       id: myRequest
-      url: https://myUrl/myPath?isMocked=true
+      headers:
+        X-Workflow-Token: A_TOKEN
+        Content-Type: application/json
+        Accept: application/json
+      body: "{\"args\": {\"content\": \"Hello world!\", \"stream\": \"A_STREAM\"}}"
       method: POST
-      body:
-        "args":
-          "message": "Hello world!"
-          "streaemId": "A_STREAM"
+      url: https://myUrl/myPath?isMocked=true
+      
+```
+
+
+```yaml
+activities:
+  - execute-request:
+      id: myRequest
       headers:
-        "X-Workflow-Token": "A_TOKEN"
+        Content-Type: multipart/form-data
+        Accept: multipart/form-data
+      body:
+        outer:
```

Comment:
does it really work with a nested structure? or does it support only one level (i.e just key-value)